### PR TITLE
feat(sensors_plus): Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by charafau**
ref: https://github.com/fluttercommunity/plus_plugins/issues/3152

Another spm support

### CODE REVIEW

**Review by miquelbeltran — COMMENTED** [review]
LGTM, still want to test on my device before approval and merge

**Review by vbuberen — APPROVED** [review]
Works good on a real device. 
Thanks for this one as well.

**Pull request merged**
